problems/strings.js

```diff
@@ -3,23 +3,29 @@
  * @param {string} - str
  * @returns {string} - first char
  */
-function firstChar() {}
+function firstChar(str) {
+  return str.slice(0, 1);
```

Comment:
This is correct but there's an easier way to do this. Think of how we get the first element of the array